src/alexapi/triggers/pocketsphinxtrigger.py

```diff
@@ -32,9 +35,8 @@ def setup(self):
 		ps_config.set_string('-keyphrase', self._tconfig['phrase'])
 		ps_config.set_float('-kws_threshold', float(self._tconfig['threshold']))
 
-		# Hide the VERY verbose logging information
-		# TODO: temporary here until the logging PR is merged - check if we need to change this
-		if not self._config['debug']:
+		# Hide the VERY verbose logging information when not in debug
+		if logger.getEffectiveLevel() != logging.DEBUG:
```

Comment:
This doesn't actually work and I wasn't able to find a working solution. Please fix. Thank you.